### ISSUE TITLE
Laser Fence

### ISSUE THREAD

**Issue by HellDragon-HK** [GreatEmerald/U2Weapons#5]
- [ ] I have witnessed several times where a bot is in the middle of a same team fence just standing there like their bot pathing broke.  On ONS-Urban I witnessed a bot just standing near a weapon locker with a fence tightly all the way around and while I'm not positive he wasn't on guard duty or actually stuck it appeared he was stuck

  - [ ] If used in DM, no laser is visible but the barrier is still there and working normally.

**Comment by GreatEmerald** [comment]
Yes, the bots have no idea how to use the field generators. I made it so that they would at least try to put them somewhere, but there's no AI code associated with them anywhere (including XMP).

**Comment by HellDragon-HK** [comment]
Anyway to at least stop them from doing it around a weapon locker or make it so the fence detects and deactivates within a larger radius?

**Comment by GreatEmerald** [comment]
No, without either writing new AI code or disabling bots' use of the field generators altogether.

**Comment by HellDragon-HK** [comment]
Darn ok then so close this one?
No chance this would cause a infinite loop crash because a bot is stuck right?

**Comment by GreatEmerald** [comment]
Not likely at all, it just makes the bot potentially stuck is all.

I might as well keep this issue around just as a point of information.
